readme.md

```diff
@@ -107,7 +113,21 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
+app.get('/song', function(req,res){
+  res.send('All song')
+})
 
+app.post('/song', function(req,res){
+  res.send('Made new song')
+})
+
+app.put('/song/:id', function(req,res){
+  res.send('Updated song')
+})
+
+app.delete('/song/:id', function(req,res){
+  res.send('Deleted song')
+})
```

Comment:
You've got the right idea, but we were asking specifically to respond with a possible RESTful action, so something like, SHOW/INDEX, UPDATE, CREATE, and DESTROY